portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -25,11 +25,7 @@
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.EntityNotFoundException;
-import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Query.FilterOperator;
 
 /* Servlet that adds a comment to the dataserve. */
 @WebServlet("/delete-comment")
```

Comment:
Don't we want to add authentication here as well? Or are you planning on doing that in a different PR?

Reply:
Planning on doing in a different PR 👍